framework/test/jest/unit/specs/application/node/transport/transport_private.spec.ts

```diff
@@ -12,38 +12,37 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-'use strict';
-
-const { TransferTransaction } = require('@liskhq/lisk-transactions');
-const { transfer, TransactionError } = require('@liskhq/lisk-transactions');
-const { validator } = require('@liskhq/lisk-validator');
-const accountFixtures = require('../../../../../fixtures//accounts');
-const { Block, GenesisBlock } = require('../../../../../fixtures//blocks');
-const {
-	Transport: TransportModule,
-} = require('../../../../../../src/application/node/transport');
-const {
-	InvalidTransactionError,
-} = require('../../../../../../src/application/node/transport/errors');
-const {
-	devnetNetworkIdentifier: networkIdentifier,
-} = require('../../../../../utils/network_identifier');
-
+import {
+	TransactionJSON,
+	TransactionError,
+	transfer,
+	TransferTransaction,
+} from '@liskhq/lisk-transactions';
+import { BlockInstance } from '@liskhq/lisk-chain';
+import { validator } from '@liskhq/lisk-validator';
+import { Logger, Channel, Processor } from '../../../../../../../src/types';
+import { Transport } from '../../../../../../../src/application/node/transport';
+
+import { genesis } from '../../../../../../fixtures/accounts';
+import { devnetNetworkIdentifier as networkIdentifier } from '../../../../../../utils/network_identifier';
+import { Block, GenesisBlock } from '../../../../../../fixtures/blocks';
+import { InvalidTransactionError } from '../../../../../../../src/application/node/transport/errors';
+/* eslint-disable @typescript-eslint/no-unnecessary-type-assertion */
```

Comment:
why this is disabled here?

Reply:
its because most of the tests in this file does not change the expression type and check the expectation of function being called with and that's why it's throwing  `This assertion is unnecessary since it does not change the type of the expression` for more than 10 tests